pkg/lib/cockpit-components-firewalld-request.scss

```diff
@@ -1,4 +1,4 @@
-@import "@patternfly/patternfly/utilities/BoxShadow/box-shadow.css";
+@use "../../node_modules/@patternfly/patternfly/utilities/BoxShadow/box-shadow.css";
```

Comment:
@use needs absolute paths when working with css otherwise it errors. I did not investigate why this difference exists, but I mention it also in the commit message.

Reply:
Ah right, sorry about that. Hard to spot the .scss vs. css difference :)